docker/.env

```diff
@@ -39,3 +37,6 @@ REACT_APP_PUBLIC_GRAPH_URI=https://localhost:8082/public
 REDIS_EVENTS_STAGING_ENDPOINT=redis:6379
 RENDER_PREVIEW=true
 SESSION_FILE_PATH_PREFIX=/tmp/
+
+# Uncomment and update ID to enable
+# DEMO_PROJECT_ID=0
```

Comment:
this would effectively disable the demo for the hobby deploy, right? i think that is the behavior we want. should we make that clear in the comment?

Reply:
Yeah, without a valid project ID it would effectively be disabled. I'll note that in the comment above.

Reply:
cool, thanks!